languages/php/example.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+require_once 'vendor/autoload.php';
+
+$access_token = '<you access token here>';
+$bitwarden_sdk = new \Bitwarden\Sdk\BitwardenSDK();
+$res = $bitwarden_sdk->authorize($access_token);
+
+// get project
+$res = $bitwarden_sdk->projectsClient->get("00056058-cc70-4cd2-baea-b0810134a729");
```

Comment:
It would be nice if the example could be run without modifying the code too much, by that I mean:
Make the user set the organization id just like we do with the access_token:
```
$access_token = '<your access token here>';
$organization_id = '<your organization id here>';
```

And then organize the requests in such a way that we don't require anything to exist. By that I mean:
- First create a project with name 'php test project' or whatever, and store it's id from the response
- Then list all projects
- Then create a secret based on the previous project id
- Then list all secrets
- Then delete the secret and project

Not a huge priority though.

Reply:
I inserted the organization as var as well. Also made similar kind of order so that nothing will be created or delete that exists outside of the example file.